R/gadgets.R

```diff
@@ -0,0 +1,58 @@
+#' Open dataset selector
+#'
+#' This function launches a shiny gadget which allows you to navigate your Crunch
+#' projects and datasets. This is useful if you can't remember a dataset's project
+#' and also saves typing long dataset names.
+#'
+#' @inheritParams listDatasets
+#' @return A `listDatasets()` call is pasted into your RStudio session`
+#' @export
+listDatasetGadget <- function(kind=c("active", "all", "archived"),
+    refresh = FALSE){
+    projects <- c("Personal Project", names(projects()))
+    personal_datasets <- listDatasets(kind = kind, refresh = refresh)
```

Comment:
You perhaps should also confirm here that the three Suggests packages this function requires are installed.